src/tests.rs

```diff
@@ -310,7 +285,7 @@ mod governor_tests {
             res.headers()
                 .get(HeaderName::from_static("x-ratelimit-remaining"))
                 .unwrap(),
-            "0" //TODO: Should this be 1?!?
+            "1" //TODO: Should this be 1?!?
```

Comment:
On my machine this test code would produce 1 instead of 0 and the TODO is kinda confusing. Please let me know if the behavior has to be as before the PR or it's ok to remove the TODO and keep the new behavior.

Reply:
I copied the test suite over from a previous project, and it returned 1. I couldn't reproduce and so changed it to 0. So you can remove the TODO and keep 1

Reply:
ok. done.